app/controllers/admin/conferences_controller.rb

```diff
@@ -61,11 +61,19 @@ def index
 
     def new
       @conference = Conference.new
+      @organizations = {}
+      Organization.all.each do |organization|
+        @organizations.store(organization.name, organization.id) if can? :create, Conference.new(organization: organization)
+      end
     end
 
     def create
-      @conference = Conference.new(conference_params)
-
+      conference_params_copy = conference_params
+      if ENV['ORGANIZATIONS_ENABLED'] != 'true'
+        org = Organization.exists?(name: 'default') ? Organization.find_by(name: 'default') : Organization.create(name: 'default')
+        conference_params_copy[:organization_id] = org.id
+      end
+      @conference = Conference.new(conference_params_copy)
```

Comment:
I was thinking we could do something simpler.
How about just finding the default organization and assigning it to the conference? We don't need to check any env variables.
Also, we know that the default organization exists, since it's [created in the migration](https://github.com/openSUSE/osem/pull/1517/files#diff-44200c9e4f9c94556ad974cc5653a0bbR15) (btw, you called it "organization", not "default").